workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -118,6 +118,8 @@ await Worker.CreateWorkerAsync(GetAlphaLocatorConfig(workerType), connectionPara
                 if (!Application.isPlaying)
                 {
                     Dispose();
+                    throw new ConnectionFailedException("Editor application stopped",
```

Comment:
What happens to the task that's being await'ed after you stop in the Editor? Is it cancelled or does it persist?

Reply:
as soon as you throw an exception, it goes into the catch block and prints an error. So it's not cancelled, but also only logs the error and doesn't do anything else anymore.

only if the application is still playing will it call `HandleWorkerConnectionFailure`,

Reply:
The task has finished by the time we get here as this is part of the continuation of 

```
 Worker = await ConnectWithRetries(connectionDelegate, MaxConnectionAttempts, logger, workerType);
```